Makefile

```diff
@@ -25,7 +25,7 @@ STATIC :=
 
 # Variables to be overridden on the command line, e.g.
 #   make test PKG=./storage TESTFLAGS=--vmodule=raft=1
-PKG          := ./...
+PKG          := $(shell go list ./... | grep -v /vendor/ | tr '\n' ' ')
```

Comment:
this will probably result in very bad performance in docker. see 53555ca14411f5bd30600e17a624080dfa93e195